### PR TITLE
Added firebase to pubspec.yaml

### PR DESCRIPTION
**Pull request by r0hnx**
Please add a google_config.json file for adding database and further development.

Refer to this link : https://firebase.flutter.dev/docs/installation/android#installing-your-firebase-configuration-file

### CODE REVIEW

**Comment by enzoftware** [comment]
You're referring to `google-services.json`?

**Comment by r0hnx** [comment]
> You're referring to `google-services.json`?

Yes

**Comment by enzoftware** [comment]
hI @r0hnx, I miss that you are assigned to the #4 issue. The idea is that everyone who wants to run the app includes their own `google-services.json` file into the project. This to avoid sharing personal sensitive data into an open-source project.

The steps you have to follow to go forward to this issue is:
- Setup your own Firebase project.
- Get the `google-services.json` from your Firebase project.
- Set up the project to run with Firebase ( Add the necessary files and dependencies to achieve that)
- Add to `.gitignore` file the `google-services.json` file path. Same for ios version.
- Implement the logic to get all data from firestore.
- Verify everything works as expected.

File: pubspec.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ dependencies:
   bordered_text: ^1.0.0
   get_it: ^4.0.4
   mockito: ^4.1.1
+  firebase_core:
+  cloud_firestore:
```
Comment (enzoftware) [inline pubspec.yaml:18]: Define specific dependency number, please.

**Comment by enzoftware** [comment]
@r0hnx Any updates?

File: pubspec.yaml (hunk 1 above)
Comment (r0hnx) [reply pubspec.yaml:18]: Its not necessary here, pub get takes the latest one automatically

**Comment by r0hnx** [comment]
> @r0hnx Any updates?

It's almost done, would push it by tomorrow

**Comment by enzoftware** [comment]
Hi @r0hnx, any update?

**Comment by enzoftware** [comment]
I'll close this PR

**Pull request closed without merging**